src/pages/task/taskTabs/buildBaronAndAnnotations/FileTicketButton.tsx

```diff
@@ -97,3 +68,5 @@ export const FileTicket: React.VFC<FileTicketProps> = ({
     </Popconfirm>
   );
 };
+
+export default FileTicketButton;
```

Comment:
unrelated nit. Would you mind adding a `width: fit-content` on the `ButtonWrapper`? The `ButtonWrapper` is too big causing stuff like this 😞

https://user-images.githubusercontent.com/47064971/166833175-1af173aa-2b8c-4add-be1f-3ccc0eb406ed.mov